### DISCUSSION TITLE
does srgb to irradiance function inversely?

### DISCUSSION THREAD

**Discussion by Jackson334-prog** [mitsuba-renderer/mitsuba3#215] [General]
I tried to write the irradiance of the projector to bitmap  Using mitsuba  Write into bitmap function. And I just use this bitmap  As another projectors irradiance bitmap,   But it seems like  This irradiance is not equal to  The original.  So I wonder how can I  Take use of it unlossness. I know that  There must be some function of srgb to irriadiance  But when I write it into bitmap  Does this function inversely?  Could you give me some advice about it?

**Comment by Speierers** [comment]
Hi @ljijjiang1234 ,

First of all you need to be careful, writing to an image file using the `Bitmap.write()` might apply some tonemapping which is probably not desired in your case. For this I suppose you take a closer look at the [how-to-guide](https://mitsuba.readthedocs.io/en/latest/src/how_to_guides/image_io_and_manipulation.html) on image manipulation in Mitsuba 3.

Moreover, when disabling the tonemapping, make sure to export to an EXR file in order to preserve the values above `1.0` in your image (e.g. HDR).

M
